client/modules/IDE/components/Editor.jsx

```diff
@@ -155,14 +160,21 @@ class Editor extends React.Component {
       }
     }, 1000));
 
-    this._cm.on('keyup', () => {
+    this.map = {};
+
+    this._cm.on('keyup', (_cm, e) => {
       const temp = this.props.t('Editor.KeyUpLineNumber', { lineNumber: parseInt((this._cm.getCursor().line) + 1, 10) });
       document.getElementById('current-line').innerHTML = temp;
+      this.handleKey(this.map, e);
     });
 
     this._cm.on('keydown', (_cm, e) => {
-      // 9 === Tab
-      if (e.keyCode === 9 && e.shiftKey) {
+      this.handleKey(this.map, e);
+      // 91 === Cmd
```

Comment:
There's a variable that's imported to this file called `metaKey` which will be set to `'Cmd'` if the user is on Mac, and `'Ctrl'` if the user is on Windows/Linux. I think you should use this variable rather than just `'Cmd'` since `'Cmd'` doesn't exist on Windows/Linux!

Reply:
Yes, I will incorporate the use of `metaKey` in my changes!